src/components/Education/Education.css

```diff
@@ -0,0 +1,25 @@
+.gray {
+  background-color: #f3f3f3;
+  padding-bottom: 10px;
```

Comment:
Set text to gray color should not change the padding-bottom.

Please ask P'Doi on how to define/styling the css krub.